resources/client/cl_match.ts

```diff
@@ -7,3 +8,11 @@ RegisterNuiProxy(MatchEvents.GET_MATCHES);
 RegisterNuiProxy(MatchEvents.SAVE_LIKES);
 RegisterNuiProxy(MatchEvents.CREATE_MY_PROFILE);
 RegisterNuiProxy(MatchEvents.UPDATE_MY_PROFILE);
+
+onNet(MatchEvents.SAVE_LIKES_BROADCAST, (result: CreateMatchBroadcast) => {
+    sendMatchEvent(MatchEvents.SAVE_LIKES_BROADCAST, result);
+});
+
+onNet(MatchEvents.CREATE_MATCH_ACCOUNT_BROADCAST, (result: FormattedProfile) => {
+    sendMatchEvent(MatchEvents.CREATE_MATCH_ACCOUNT_BROADCAST, result);
+});
```

Comment:
What's the reasoning behind needing a match account broadcast event? I believe we are doing active data fetching from NUI to receive pending matches on Match open. If I recall correctly, this shouldn't need to be pushed from the server.

Reply:
Exactly, when the match app is open. But if you open the app and then another persons account is created, that persons account is not gonna appear to you. This is like twitter, when a new tweet is created that tweet is send to the other player.

Reply:
Yes but for what reason do we need to push new accounts, this should be all handled on the server side?

Reply:
Any idea other than that? Because, I only see giving the players the new account. Only if we use emitOnOpen so that it only requests when the app is open.

Reply:
emitOnOpen was primarily used in the past to signal to game scripts fetching needs to occur. Since then, we have taken an NUI primary approach where data fetching is generally handled through the proxy system and is started from CEF.

Reply:
So do you have another approach that makes the same effect?

Reply:
The personal approach I would suggest is offloading matching in some server side handler. Then, whenever a client needs to request matches. It can do a simple fetch to a promise event on the server to get all matches when needed.

Reply:
Ok, but when is that event is going to be triggered? How is the NUI going to figure out that a new possible match exists without going to check first?

Reply:
> Ok, but when is that event is going to be triggered? How is the NUI going to figure out that a new possible match exists without going to check first?

I'll need to review the match application in more depth, but from an architecture perspective we are attempting to reduce fire-and-forget game events events and are looking to replicate web behavior somewhat for data fetching.